snapcraft/parts/lifecycle.py

```diff
@@ -506,9 +506,9 @@ def _run_in_provider(
 
     if getattr(parsed_args, "enable_manifest", False):
         cmd.append("--enable-manifest")
-    build_information = getattr(parsed_args, "manifest_build_information", None)
-    if build_information:
-        cmd.extend(["--manifest-build-information", build_information])
```

Comment:
so is `--manifest-build-information` a completely invalid/inexisting flag?

Reply:
Good catch, this was likely a mix-up with `SNAPCRAFT_BUILD_INFO`. Can we have a spread test using the option in non-destructive-mode?

Reply:
You already wrote a spread test it with LXD and destructively!